server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
 
 Existing user::
 
-When existing users try to log in, they are required to register their WebAuthn authenticator automatically :
+In case that `WebAuthn Authenticator` is setup as required as shown in the first example, then when existing users try to log in,
+they are required to register their WebAuthn authenticator automatically :
 
 - Open the login form.
 - Fill in items, click `Save` and  click `Login`.
 - When the users log in, they are required to register their WebAuthn authenticator.
 - After successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label.
 
-===== View Registered WebAuthn Authenticator
+==== Passwordless WebAuthn
+
+WebAuthn is often used for two-factor authentication, however it can be desired to use it also as first factor authentication. In this case,
+user with `passwordless` WebAuthn credential will be able to authenticate to {project_name} without password. {project_name} allows to use WebAuthn
+as both passwordless and two-factor authentication mechanism in the context of single realm and even in the context of single authentication flow.
+
+Administrator may typically require that Security Keys registered by users for the WebAuthn passwordless authentication must meet different
+(usually stronger) requirements. For example those security keys may require users to authenticate to that security key using PIN, or the
+security key should be attested with stronger certificate authority.
+
+Because of this, {project_name} allows administrator to configure separate `WebAuthn Passwordless Policy`. There is also separate required action
+of type `Webauthn Register Passwordless` and separate authenticator of type `WebAuthn Passwordless Authenticator`.
+
+===== Setup
+
+The setup procedure of WebAuthn passwordless support is the following :
+
+* Register new required action for WebAuthn passwordless support. Use the same steps as described <<_webauthn-register, above>>
+with the only difference, that you need to register action called `Webauthn Register Passwordless`.
+
+* Configure the policy. You can use same steps and configuration options as described <<_webauthn-policy, above>>, however you
+need to configure them in the admin console in the tab `WebAuthn Passwordless Policy`. You can configure this policy as you want, however
+typically the requirements for security key will be stronger than for two-factor policy. For example `User Verification Requirement` can
+be set to `Required` when configure passwordless policy.
+
+* Finally configure the authentication flow. Let's assume that we will use same flow called `WebAuthn Browser` as described
+<<_webauthn-authenticator-setup, above>>, but
+we will configure it like this:
+
+** The `WebAuthn Browser Forms` subflow will contain `Username Form` as first authenticator. This means that user will provide just
```

Comment:
add "the" before "first"
This setting means that the user will provide just his or her username as the first step.